openlibrary/plugins/openlibrary/tests/test_lists.py

```diff
@@ -72,12 +72,12 @@ def test_from_input_with_json_data(self):
                 seeds=[{'key': '/books/OL1M'}, {'key': '/books/OL2M'}],
             )
 
-    SEED_TESTS = [
+    SEED_TESTS = (
```

Comment:
This one is apparently necessary :P
```suggestion
    SEED_TESTS: tuple = (
```